packages/client/webapp/tests/e2e/base.spec.ts

```diff
@@ -19,4 +19,9 @@ test.describe('main landing page test', () => {
 		// Expect an attribute "to be strictly equal" to the value.
 		await expect(header).toBeVisible();
 	});
+
+	test('should not display sidebar on home page', async ({ page }) => {
+		const sidebar = page.locator('data-test-id=sidebar');
```

Comment:
The classname `sidebar` did not work for the test.

Reply:
Selecting based on the class `sidebar` works but I figured class implies it may not be unique.